blt/src/Commands/HumsciCommand.php

```diff
@@ -2,15 +2,25 @@
 
 namespace Acquia\Blt\Custom\Commands;
 
-use Acquia\Blt\Robo\BltTasks;
 use Acquia\Blt\Robo\Commands\Artifact\AcHooksCommand;
 use Acquia\Blt\Robo\Exceptions\BltException;
+use Consolidation\AnnotatedCommand\CommandData;
 
 /**
  * Defines commands in the "custom" namespace.
  */
 class HumsciCommand extends AcHooksCommand {
 
+  /**
+   * Log in when drupal:sync finishes.
+   *
+   * @hook post-command drupal:sync
+   */
+  public function postArtifactBuild($result, CommandData $commandData) {
```

Comment:
PostArtifactBuild doesn't sound like the end of drupal:sync. It sounds like what happens after blt:deploy.

Reply:
ooh yeah.. copy paste bad mike